src/swap/components/SwapProvider.tsx

```diff
@@ -56,6 +56,7 @@ export function SwapProvider({
     statusName: 'init',
     statusData: {
       isMissingRequiredField: true,
+      maxSlippage: experimental.maxSlippage ?? 3,
```

Comment:
Btw, in general I don't understand why you are touching `SwapProvider` for a PR that is only about Swap Input.

You should be able to write the Input logic and Input tests in isolation without the need to modify the `Swap Provider`.

Touching the Swap Provider, will just delay this PR to be merged.

My advice is to not touch the Swap Provider in this PR and just focus on Input.